ably.d.ts

```diff
@@ -2694,32 +2710,32 @@ declare namespace Types {
      *
      * @param event - The event name.
      * @param listener - An event listener function.
-     * @returns A promise which resolves upon success of the channel {@link RealtimeChannelPromise.attach | `attach()`} operation and rejects with an {@link ErrorInfo} object upon its failure.
+     * @returns A promise which, upon successful attachment to the channel, will be fulfilled with a {@link ChannelStateChange} object. Upon failure, the promise will be rejected with an {@link ErrorInfo} object.
```

Comment:
This one is still missing `null` explanation?

Reply:
fixed